Lovers Quarrel: Version 1.010; ttfautohint (v1.8.3) added

vv-monsalve opened the pull request:
5808fe6: [gftools-packager] Lovers Quarrel: Version 1.010; ttfautohint (v1.8.3) added

* Lovers Quarrel Version 1.010; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/googlefonts/lovers-quarrel at commit https://github.com/googlefonts/lovers-quarrel/commit/de6ca22b9f03fa8436d25f5f09d8ac3288c80aa0.

 2ef78a6: [gftools-packager] ofl/loversquarrel remove METADATA "source".  google/fonts#2587

m4rc1e commented on ofl/loversquarrel/DESCRIPTION.en_us.html:
```diff
@@ -1 +1,10 @@
-<p>Lovers Quarrel is an ornate calligraphic script. It has beautifully embellished uppercase characters and clean, legible lowercase forms.</p> <p>Lovers Quarrel is a Unicode typeface that supports languages that use the Latin script and its variants, and could be expanded to support other scripts. To contribute to the project contact <a href="mailto:typesetit@att.net">Rob Leuschke</a>.</p>
\ No newline at end of file
+<p>
+MonteCarlo is a beautiful formal script— both contemporary and traditional. This connecting script’s italic is slight, making it an extremely legible design.
```
I think you accidentally copied the wrong description

vv-monsalve replied:
No, I forgot to save and push the latest changes.

m4rc1e reviewed (changes requested):
Everything look good apart from the description. Feel free to merge once it has been edited.

vv-monsalve commented:
Updated

## Lovers Quarrel: Version 1.010; ttfautohint (v1.8.3) added

---
 6840db8: [gftools-packager] Lovers Quarrel: Version 1.010; ttfautohint (v1.8.3) added

* Lovers Quarrel Version 1.010; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/googlefonts/lovers-quarrel at commit https://github.com/googlefonts/lovers-quarrel/commit/bf2edcda853feb2fffc591ee9870c0be6198779b.

 b7552ad: [gftools-packager] ofl/loversquarrel remove METADATA "source".  google/fonts#2587

vv-monsalve commented:
> Everything look good apart from the description. Feel free to merge once it has been edited.

Thanks. I'll do so.

vv-monsalve commented:
Updated

## Lovers Quarrel: Version 1.010; ttfautohint (v1.8.3) added

---
 6031281: [gftools-packager] Lovers Quarrel: Version 1.010; ttfautohint (v1.8.3) added

* Lovers Quarrel Version 1.010; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/googlefonts/lovers-quarrel at commit https://github.com/googlefonts/lovers-quarrel/commit/bf3a5fd5bf11cd96b5545ee9191a16460296eb71.

 bff80b4: [gftools-packager] ofl/loversquarrel remove METADATA "source".  google/fonts#2587
